imports/plugins/core/ui/client/components/link/link.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+export default class Link extends Component {
```

Comment:
If this is something we should be using elsewhere do we need to add some docs on it, or at least a JSDoc?

Reply:
@zenweasel I just added a JSDoc and created a docs page PR: https://github.com/reactioncommerce/reaction-docs/pull/653